### PR TITLE
core: frontend: views: NetworkTestView: Move from await to finally

### PR DESCRIPTION
**Pull request by patrickelectric**
Signed-off-by: Patrick José Pereira <patrickelectric@gmail.com>

### CODE REVIEW

File: core/frontend/src/views/NetworkTestView.vue [hunk 1]
```diff
@@ -181,7 +181,7 @@ export default Vue.extend({
     async checkUploadSpeed(): Promise<void> {
```
Comment (rafaellehmkuhl) [inline core/frontend/src/views/NetworkTestView.vue:181]: In this case, you don't need to declare the method as asynchronous, right?

**Review by rafaellehmkuhl — CHANGES_REQUESTED** [review]

**Review by Williangalvani — APPROVED** [review]
looks good, but there's a typo in the commit msg

**Comment by patrickelectric** [comment]
> looks good, but there's a typo in the commit msg

done.

**Comment by rafaellehmkuhl** [comment]
Just tested. Everything working as expected.

**Pull request merged**